conf/messages

```diff
@@ -28,7 +28,7 @@ what-is-your-national-insurance-number.nino.title=Enter your National Insurance
 what-is-your-national-insurance-number.nino.label=Enter your National Insurance number
 what-is-your-national-insurance-number.nino.hint=It’s on your National Insurance card, benefit letter, payslip or P60. For example, ‘QQ 12 34 56 C’.
 what-is-your-national-insurance-number.nino.invalid=Enter your National Insurance number
-what-is-your-national-insurance-number.nino.unavailable=I do not have a National Insurance number, or have lost or forgotten it
+what-is-your-national-insurance-number.nino.unavailable=<a href="{0}" class="govuk-link">I do not have a National Insurance number, or have lost or forgotten it</a>
```

Comment:
Should be be putting html in messages? Is there a reason to not put it in the view?

Reply:
I have removed the html from this message by using the link component instead